ibis/spark/tests/test_client.py

```diff
@@ -0,0 +1,42 @@
+import ibis
+
+
+def test_list_databases(client):
+    dbs = client.list_databases()
+    assert dbs == ['default']
+
+
+def test_list_tables(client):
+    tables = client.list_tables()
+    assert tables == ['nested_types', 'simple']
+
+
+def test_get_schema(client):
+    schema = client.get_schema('simple')
+    assert schema == ibis.schema([('foo', 'int64'),
+                                  ('bar', 'string')])
+
+
+def test_table_simple(client):
+    db = client.database()
+    correct_string = ''.join([
+        'SparkTable[table]\n  ',
+        'name: simple\n  schema:\n    ',
+        'foo : int64\n    bar : string'
+    ])
+    assert str(client.table('simple')) == correct_string
```

Comment:
This is a bit fragile, since the `str`/`repr` of a thing is not really guaranteed to be anything except that calling it shouldn't raise an exception.

It's better if you test some attributes of the table, and something like `assert isinstance(table, ir.TableExpr)` and `assert isinstance(table.op(), SparkTable)`